docs/specification-ligo.md

```diff
@@ -156,19 +156,19 @@ Each address can have any number of operators and be an operator of any number o
 ## Proposals
```

Comment:
I suggest to add some info about the stages here and remove some details from below as a consequence:
```suggestion
## Stages

The contract constantly cycles between two stages, a proposing stage and a voting stage.
Both have the same same length, `voting_period` and alternate between each other,
starting from "voting" for period number `0`.

Tokens can be frozen in any period, but they can only be used for voting, proposing
and unfreezing starting from the following one and onwards.

### Proposals
```